lib/review.lib.js

```diff
@@ -0,0 +1,84 @@
+const Review = require('../models/review.model');
+const ErrorResponse = require('../utils/errorResponse.util');
+const BookLib = require('./book.lib');
+const UserLib = require('./user.lib');
+
+class ReviewLib {
+  constructor() {
+    this.ReviewModel = Review;
+    this.bookLib = new BookLib();
+    this.userLib = new UserLib();
+  }
+
+  checkResourceInDB = async ({ user, book }) => {
+    if (user) {
```

Comment:
Consider renaming this method to something like checkUserAndBookInDB.

Reply:
I actually thought that it would be too long